spec/datadog/ci/ext/fixtures/ci/github.json

```diff
@@ -15,7 +42,7 @@
       "ci.pipeline.id": "ghactions-pipeline-id",
       "ci.pipeline.name": "ghactions-pipeline-name",
       "ci.pipeline.number": "ghactions-pipeline-number",
-      "ci.pipeline.url": "https://github.com/ghactions-repo/commit/ghactions-commit/checks",
+      "ci.pipeline.url": "https://github.com/ghactions-repo/actions/runs/ghactions-pipeline-id/attempts/ghactions-run-attempt",
```

Comment:
Can you confirm this is expected?

E.g. the git commit only mentions the introduction of `env['GITHUB_SERVER_URL']` but there's a different feature being changed here, which is changing the pipeline url.

Reply:
yeah sry about that. I tend to always squash my PRs so I pay more attention to the PR description than the commit messages. This change is reflected in the description but not in the commit message.

Reply:
Ah, right. I totally missed it.